setup.py

```diff
@@ -1,5 +1,9 @@
 from setuptools import setup, find_packages
-import pathlib
+
+try:
+    import pathlib
+except ImportError:
+    import pathlib2 as pathlib
 
 here = pathlib.Path(__file__).parent
```

Comment:
change this line to use `os` instead
```suggestion
here = os.path.abspath(os.path.dirname(__file__))
```